server/schedulers/hot_region.go

```diff
@@ -244,6 +244,10 @@ func (h *hotScheduler) summaryPendingInfluence() {
 // ended operator
 func (h *hotScheduler) gcRegionPendings() {
 	for regionID, op := range h.regionPendings {
+		if op == nil {
+			delete(h.regionPendings, regionID)
```

Comment:
When will it happen?

Reply:
This was the case in the previous test case; btw, this pr #3658 3658 will only involve the refactoring of pending influence